### DISCUSSION TITLE
output in scientific notation with a power of ten instead of E-notation

### DISCUSSION THREAD

**Discussion by eadmaster** [Qalculate/libqalculate#614] [Q&A]
e.g.:
````
qalc -s "exp mode 3" "3000*(3000-1) / 2"

(3000 × (3000 − 1)) / 2 = 4.4985E6
````

i'd like to get instead:
`````
(3000 × (3000 − 1)) / 2 = 4.4985 * 10^6
`````

**Comment by hanna-kn** [comment]
An option to disable e-notation (enabled 10^x notation) has now been added (https://github.com/Qalculate/libqalculate/commit/b12d51a29d4250a729ac5f46d44936b3d80c38a4).

With the new option the following is possible.
```
qalc -s "e notation 0" -s "exp mode 3" "3000*(3000-1) / 2"
(3000 × (3000 − 1)) / 2 = 4.4985 × 10^6
```

**Comment by eadmaster** [comment]
thanks, i was expecting another "exp mode X" value, but this will do the job as well!

**Comment by eadmaster** [comment]
as alternative, i've made this sed script:
````
sed 's/\([0-9]*.[0-9]*\)E\([0-9]*\)/\1 * 10^\2/g'
````
